packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -788,6 +794,82 @@ describe('Dropdown', () => {
         ),
       ).toBeTruthy()
     })
+
+    it('has the onAdd message inserted after an item has been added to selection', () => {
+      const wrapper = mountWithProvider(
+        <Dropdown
+          multiple
+          items={items}
+          getA11ySelectionMessage={{ onAdd: item => 'bla bla added' }}
+        />,
+      )
+      const dropdown = wrapper.find(Dropdown)
+      const triggerButton = wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`)
+
+      triggerButton.simulate('click')
+      const firstItem = wrapper.find(`li.${Dropdown.slotClassNames.item}`).at(0)
+      firstItem.simulate('click')
+
+      expect(dropdown.state('a11ySelectionStatus')).toBe('bla bla added')
+    })
+
+    it('has the onRemove message inserted after an item has been removed from selection', () => {
+      const wrapper = mountWithProvider(
+        <Dropdown
+          multiple
+          items={items}
+          getA11ySelectionMessage={{ onRemove: item => 'bla bla removed' }}
+        />,
+      )
+      const dropdown = wrapper.find(Dropdown)
+      const triggerButton = wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`)
+
+      triggerButton.simulate('click')
+      const firstItem = wrapper.find(`li.${Dropdown.slotClassNames.item}`).at(0)
+      firstItem.simulate('click')
+      jest.runAllTimers()
+      const removeIcon = wrapper.find(`span.${DropdownSelectedItem.slotClassNames.icon}`)
+      removeIcon.simulate('click')
+
+      expect(dropdown.state('a11ySelectionStatus')).toBe('bla bla removed')
+    })
+
+    it('has the onAdd message cleared after being displayed', () => {
+      const wrapper = mountWithProvider(
+        <Dropdown multiple items={items} getA11ySelectionMessage={{ onAdd: item => 'bla bla' }} />,
+      )
+      const dropdown = wrapper.find(Dropdown)
+      const triggerButton = wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`)
+
+      triggerButton.simulate('click')
+      const firstItem = wrapper.find(`li.${Dropdown.slotClassNames.item}`).at(0)
+      firstItem.simulate('click')
+
+      jest.runAllTimers()
+      expect(dropdown.state('a11ySelectionStatus')).toBe('')
```

Comment:
Can we merge these suites with previous?

```js
firstItem.simulate('click')
expect(dropdown.state('a11ySelectionStatus')).toBe('bla bla added')

jest.runAllTimers()
expect(dropdown.state('a11ySelectionStatus')).toBe('')
```

Reply:
did, thanks !